python/tvm/driver/tvmc/common.py

```diff
@@ -333,6 +333,29 @@ def tracker_host_port_from_cli(rpc_tracker_str):
     return rpc_hostname, rpc_port
 
 
+def parse_disabled_pass(input_string):
```

Comment:
This utility could be more general. Since its functionatlity is taking a list of passes in a string and checking if they are valid, we don't need to limit this function to "disabled" passes. Instead, we could name it `parse_pass_list_str`, which does the same thing as now.